docs/applications/nube-sdk/slots/storefront-slots.md

```
@@ -66,20 +66,53 @@ Product grid slots are special slots that allow you to render components on prod
-      <Button onClick={() => console.log(product)}>
-        Click Me!
-      </Button>
-    </Box>
-  ));
-});
+import { Text } from "@tiendanube/nube-sdk-jsx";
+import type { NubeSDK, ProductDetails } from "@tiendanube/nube-sdk-types";
+import { styled } from "@tiendanube/nube-sdk-ui";
+
+const StyledText = styled(Text)`
+	color: red;
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)